utils/OData.js

```diff
@@ -301,6 +307,9 @@ OData.prototype._post = function (data) {
   return new Promise((resolve, reject) => {
     const mainFields = modelUtils.getMainFields(this.name)
     data = R.pick(mainFields, data)
```

Comment:
This row already filters the data and on the next row the dublicate field is missing.